lib/rspec/rails/example/system_example_group.rb

```diff
@@ -78,16 +78,19 @@ def app
         def initialize(*args, &blk)
           super(*args, &blk)
           @driver = nil
+
+          self.class.before do
+            # A user may have already set the driver, so only default if driver
+            # is not set
+            driven_by(:selenium) unless @driver
+          end
         end
 
```

Comment:
Thank you for your suggestion.
but I think this suggest is a little complicated.

This problem is due to the execution order of before process.
For this reason, I think it's easier to use `self.class.before` rather than introducing new `@driver_args` and `@driver_blk`.

Reply:
I prefer it as it is now too FWIW